BUILD/settings/any.dat

```diff
@@ -54,6 +54,7 @@ auto_saveMagicalSausage	boolean	When true, don't eat magical sausage, so you can
 auto_useWishes	boolean	When true, use the Genie Bottle to go faster in non-Community Service runs.
 auto_spoonsign	string	What sign to change to with the hewn moon-rune spoon after finishing any business in the current sign. If blank or invalid, we won't switch automatically. Can be set to any of the 9 main sign names, or knoll/canadia/gnomad to automatically select the appropriate sign within that zone based on your mainstat. Also some shorthands are allowed in case you forgot which moonsign is which, such as famweight/clover/food/booze. You will be prompted to confirm that you actually do want to change signs automatically once per ascension (although it will automatically assume that you do after 15 seconds of inactivity), so forgetting to change the value in advance should not be a concern.
 auto_MLSafetyLimit	integer	If set this will be the (approximate) cap for +ML. WARNING: Certain conditions may require the script to exceed this by a small margin. For best results set this 10 below the maximum ML you can handle.
+auto_ignoreL13Slowdown	boolean	When true, the script will not scale back ML after reaching Level 13. WARNING: You are unlikely to get Instant Karma (This is useful if your run preceeds The Sea, The Basement, or Dungeons)
```

Comment:
bike-shedding, but "L13 slowdown" feels a bit unclear. Perhaps `auto_skipInstantKarma` or `auto_runMLAfter13`?

Reply:
changed to `auto_disregardInstantKarma`